gui/widgets/qopenglmousetracker.cpp

```diff
@@ -21,8 +21,11 @@ QOpenGLMouseTracker::QOpenGLMouseTracker(QWidget *parent, float lineWidth,
       m_lastColour(QWidget::palette().color(QWidget::backgroundRole()).rgb()) {}
 
 std::shared_ptr<rendering::ClippingPlane>
-QOpenGLMouseTracker::BuildClippingPlane(GLfloat a, GLfloat b, GLfloat c,
-                                        GLfloat d, bool active) {
+QOpenGLMouseTracker::BuildClippingPlane(
+    GLfloat a, GLfloat b, GLfloat c, GLfloat d, bool active,
+    std::shared_ptr<rendering::Node> parent) {
+
+  assert(parent == nullptr);
```

Comment:
what is the unused parent argument for here? also this function doesn't appear to be used, can it be removed?